packages/js/app/package.json

```diff
@@ -20,6 +20,8 @@
         "@react-native-community/masked-view": "*",
         "@react-navigation/bottom-tabs": "6.3.1",
         "@react-navigation/native": "6.0.10",
+        "@react-navigation/native-stack": "^6.7.0",
+        "@rnmapbox/maps": "rnmapbox/maps#main",
```

Comment:
What is this dependency?

Reply:
Used to create the props that allow you to navigate and route

Reply:
Ignore my message before if you saw it
You're right, I should just be using @react-navigtion/stack and get rid of unnecessary dependency